src/averageAges.js

```diff
@@ -37,7 +47,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const selection = people.filter(woman => {
```

Comment:
is this woman who stored inside this variable?

Reply:
👌fixed